atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2566,8 +2551,7 @@ private void handleKeyAlreadyExistsException(
             }
 
             SummarizedLockCheckResult lockCheckResult =
-                    immutableTimestampLockManager.getExpiredImmutableTimestampAndCommitLocksWithFullSummary(
-                            commitLocksToken);
+                    transactionLocksManager.getExpiredImmutableTimestampAndCommitLocksWithFullSummary();
```

Comment:
Call is equivalent to passing it explicitly here.

Reply:
Agreed.